src/config/configuration.ts

```diff
@@ -39,15 +39,15 @@ export default (): any => {
         email: {
             host: process.env.EMAIL_HOST || "smtp.ethereal.email",
             port: process.env.EMAIL_PORT || 587,
-            user: process.env.EMAIL_USER || "ara.kertzmann8@ethereal.email",
```

Comment:
Was this changed on purpose?

Reply:
Yes.

ethereal.email uses auto-generated accounts, and are only used to test mails during development. The account in the config did not work anymore, so I auto-generated a new one, and left it in the config, to make it (very slightly) easier for the next task